ApiSurface/ApiSurface.fsproj

```diff
@@ -35,7 +35,7 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="FSharp.Core" Version="4.3.4" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="System.Text.Json" Version="6.0.0" />
```

Comment:
This is the earliest acceptable version. In v5, it failed to serialise/deserialise F# records.

Reply:
Can you use a lower version and use `[<CLIMutable>]` on the record instead?

Reply:
No.

```
The collection type 'Microsoft.FSharp.Collections.FSharpList`1[System.String]' is abstract, an interface, or is read only, and could not be instantiated and populated. Path: $.publicReleaseRefSpec | LineNumber: 2 | BytePositionInLine: 27.
```

(tested in https://github.com/G-Research/ApiSurface/pull/23)